api/kyverno/v1/image_verification_types.go

```diff
@@ -305,7 +309,9 @@ func (ska *StaticKeyAttestor) Validate(path *field.Path) (errs field.ErrorList)
 	if ska.PublicKeys == "" {
 		errs = append(errs, field.Invalid(path, ska, "A key is required"))
 	}
-
+	if ska.PublicKeys != "" && ska.SignatureAlgorithm != "" && ska.SignatureAlgorithm != "sha256" && ska.SignatureAlgorithm != "sha512" {
```

Comment:
```ska.SignatureAlgorithm != "" ```
Is this check required? I think you mentioned that it has default value of sha256, so it'll never be empty string.

Reply:
yes. will remove it.